### PR TITLE
condense fluent-bit config into single file and reduce number of cont…

### PR DESCRIPTION
**Pull request by jim-minter**
…ainers

### Which issue this PR addresses:

n/a

### What this PR does / why we need it:

condense fluent-bit config into single file and reduce number of containers.  Minor optimisation.

### Test plan for issue:

Will test this manually (not done yet)

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

**Comment by jim-minter** [comment]
tested manually

**Review by ihcsim — APPROVED** [review]
LGTM - nice cleanup!

File: pkg/operator/controllers/genevalogging/genevalogging.go [hunk 1]
```diff
@@ -118,88 +118,14 @@ func (g *GenevaloggingReconciler) daemonset(cluster *arov1alpha1.Cluster) (*apps
...
-								},
-								{
-									Name:      "log",
-									ReadOnly:  true,
-									MountPath: "/var/log",
-								},
-								{
-									Name:      "fluent",
-									MountPath: "/var/lib/fluent",
-								},
```
Comment (mjudeikis) [inline pkg/operator/controllers/genevalogging/genevalogging.go:191]: Is this not needed anymore?

**Comment by mjudeikis** [comment]
O_O When you see it. Why we had this in the first place?

**Review by mjudeikis — APPROVED** [review]

**Pull request merged**